DxDispatch/src/model/JsonParsers.cpp

```diff
@@ -1468,6 +1467,23 @@ Model::DmlDispatchableDesc ParseModelDmlDispatchableDesc(const rapidjson::Value&
     return desc;
 }
 
+Model::DmlSerializedGraphDispatchableDesc ParseModelDmlSerializedGraphDispatchableDesc(const std::filesystem::path& parentPath, const rapidjson::Value& object)
+{
+    Model::DmlSerializedGraphDispatchableDesc desc = {};
+    desc.sourcePath = ResolveInputFilePath(parentPath, ParseStringField(object, "sourcePath"));
+    desc.executionFlags = ParseDmlExecutionFlagsField(object, "executionFlags", false, DML_EXECUTION_FLAG_NONE);
+
+    auto bindingsField = object.FindMember("bindings");
+    if (bindingsField != object.MemberEnd() && bindingsField->value.IsObject())
+    {
+        for (auto bindingMember = bindingsField->value.MemberBegin(); bindingMember != bindingsField->value.MemberEnd(); bindingMember++)
+        {
+            desc.initBindings[bindingMember->name.GetString()] = ParseBindingSource(bindingMember->value);
+        }
+    }
```

Comment:
This is exactly same as DmlDispatchable. So can we move this logic into a separate local function and call it in both place.